src/frontend/Komponenter/Behandling/Totrinnskontroll/SendTilBeslutterFooter.tsx

```diff
@@ -83,13 +133,26 @@ const SendTilBeslutterFooter: React.FC<{
         <>
             {behandlingErRedigerbar && (
                 <Footer>
+                    {kanOppretteFremleggsoppgave && (
+                        <Alertstripe variant={'info'} fullWidth={true}>
+                            Det blir automatisk opprettet oppgave for kontroll av
```

Comment:
```suggestion
                            <BodyShort> Det blir automatisk opprettet oppgave for kontroll av
```
Wrap tekst i BodyShort og fjern mellomrommet på slutten